app/views/stories/_main_stories_feed.html.erb

```diff
@@ -19,3 +19,89 @@
     <div class="single-article-small-pic" id="article-index-hidden-div" style="display:none"></div>
   <% end %>
 </div>
+
+<script>
```

Comment:
Hmmmmm I'm not sure about plopping all of this in the HTML like this.

If so, let's add `defer` just because we typically defer JS tags, but yeah @nickytonline if you want to give more thoughts on the general arrangement here?

Reply:
100% agree on deferring the tag.

I would honestly rather have this code elsewhere, but it's so specific to the article feed (while logged off), and none of the code should be reused.

Feedback is welcome on how to handle the differences between:
1. logged off article feed in Rails
2. logged in article feed in preact
3. There are also differences in the structure of the HTML between the two